monitoring/ceph-mixin/README.md

```diff
@@ -0,0 +1,50 @@
+## Prometheus Monitoring Mixin for Ceph
+A set of Grafana dashboards and Prometheus alerts for Ceph.
+
+All the Grafana dashboards are already generated in the `dashboards_out`
+directory and alerts in the `prometheus_alerts.yaml` file.
+
+You can use the Grafana dashboards and alerts with Jsonnet like any other
+prometheus mixin. You can find more ressources about mixins in general on
+[monitoring.mixins.dev](https://monitoring.mixins.dev/).
+
+### Grafana dashboards for Ceph
+In `dashboards_out` you can find a collection of
+[Grafana](https://grafana.com/grafana) dashboards for Ceph Monitoring.
+
+These dashboards are based on metrics collected
+from [prometheus](https://prometheus.io/) scraping the [prometheus mgr
+plugin](http://docs.ceph.com/en/latest/mgr/prometheus/) and the
+[node_exporter](https://github.com/prometheus/node_exporter).
+
+#### Requirements
+
+- [Status Panel](https://grafana.com/plugins/vonage-status-panel) installed on
+  your Grafana instance
```

Comment:
The `grafana-piechart-panel` was also required. Is it by default now included?

Reply:
Looking at this again apparently there is now a pie chart panel by default in grafana (see this commit https://github.com/grafana/piechart-panel/commit/b4719bfa0452cf22661d7262bfe736af157dfd1c#diff-81bc52c095e350118867727ac555c181ec0e4b8de06521f704677f7a60cdb5b8R5). Not sure if it is compatible directly though, I will check this!

Reply:
Great! If we could get rid of the status panel by using some other built-in chart (any clue?), we could replace our custom Grafana image with the vanilla one.

It seems that there are a couple of built-in widgets that could potentially fit here (some of them have been introduced in Grafana 8.x):
* https://grafana.com/docs/grafana/latest/visualizations/status-history/
* https://grafana.com/docs/grafana/latest/visualizations/state-timeline/

Reply:
I think I will just add back the requirement in the README and check how we could use the builtin type in another PR.